### ISSUE TITLE
Page does not display properly - raw text returned in Safari

### ISSUE THREAD

**Issue by zedwick** [Rsl1122/Plan-PlayerAnalytics#54]
**Plan Version:** 2.8.4
**Server Version:** git-Paper-1064 (MC: 1.11.2)
**Database Type:** sqlite
**Command Causing Issue:** /plan analyze

**Description:**
Loading the site linked after clicking "click me" from the ```/plan analyse``` command returns raw text in Safari. However, I can confirm that it works fine under Chrome for me.

**Steps to Reproduce:**

1. ```/plan analyse``` in-game
2. Click "Click me!" to load website
3. Load link in Safari

**Snippet from returned webpage:**
```
HTTP/1.1 OK
Content-Type: text/html; charset=utf-8
Content-Length: 40448

<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Plan | Server Analysis</title>
        <meta name="description" content="Player Analysis window">
        <meta name="author" content="Rsl1122">
```

The above is taken from the top of the returned webpage.

**Comment by Rsl1122** [comment]
What version of safari are you using?

**Comment by zedwick** [comment]
I'm using Safari 9.1.1

**Comment by Rsl1122** [comment]
I found this, it might solve the issue

> With the same problem, here is what worked for me (Sarfari Version 7.1.8):
> In a nutshell: under the Develop drop down menu, uncheck " Disable Styles (and uncheck Disable Images if it is checked)"
> This worked for me, every time.
>  
> Here are the steps to get there:
> Under Safari Go to 3rd line: Safari Preferences
> In the Popup, Go to Advanced; In the last line check "Show Develop in menu bar"
> Then; on the Safari top bar, go to Develop (between Bookmarks & Window)
> In the Develop drop down, click "Empty Caches" (same as Command+Option+e)
> Then; press Develop again and in the drop down, uncheck "Disable Styles" and also uncheck Disable Images if it is checked)

https://discussions.apple.com/thread/2725780?tstart=0

**Comment by zedwick** [comment]
Neither of those are currently disabled. I emptied the cache anyway, but the problem persists.

**Comment by zedwick** [comment]
Unfortunately I believe Apple discontinued Safari for windows a couple of years ago.

I have put those files into the plan directory, removed the first line and can confirm it did use the file but did not resolve the issue.

**Comment by zedwick** [comment]
I've worked out that accessing the root (```ip:port/```) page shows correctly, as does the "404 Data was not found in cache" page at ```/securityCode/player/Zedwick```. Looking at your code, the only difference I can see is ```Content-Type: text/html\r\n``` works where-as ```Content-Type: text/html; charset=utf-8\r\n``` does not, for whatever reason.

**Comment by zedwick** [comment]
Nope, I think it might be your first line.
```HTTP/1.1 OK``` should be ```HTTP/1.1 200 OK```

**Comment by Rsl1122** [comment]
Alright I'll try changing things and upload a test fix to the test server today. It has the analysis page always up so it should be perfect for seeing if it fixes things.

**Comment by Rsl1122** [comment]
Alright try going to http://137.74.196.227:8804/bAkEd/server in safari and see if it works now. I've changed the response:

> HTTP/1.1 OK should be HTTP/1.1 200 OK

**Comment by zedwick** [comment]
Yeah, that works fine in Safari!

**Comment by Rsl1122** [comment]
Alright, thanks for helping me fix it :) 
Fix will be applied in 2.9.0 (after I finish the demographics section on analysis page)

**Comment by zedwick** [comment]
Awesome, thanks!
